### DISCUSSION TITLE
Request for an English template

### DISCUSSION THREAD

**Discussion by jomacezu** [windingwind/zotero-better-notes#1227] [Q&A]
Hi, thank you for the great work on this plugin. I wasn’t sure where to ask this question, so I decided to post it here.

I am looking for a functional note template in English for taking notes on items in Zotero using Better Notes. I have noticed that many shared templates, as well as the solutions to template errors, are in Chinese. This makes it hard for me to understand, as I don’t speak Chinese.

If you could share a tested and working template in English, I would greatly appreciate it. Also, if the solution lies in a discussion written in Chinese, I would be grateful if you could point out the exact part where the functional code is provided.

Thank you in advance for your help.

I am using Zotero 7.

I am currently using version 2.1.9 of the plugin.

**Comment by windingwind** [comment]
What kind of template are you looking for?

**Reply by jomacezu** [reply]
I am looking for a template to take notes on scientific articles, similar to the one shared in [this discussion](https://github.com/windingwind/zotero-better-notes/discussions/771). However, I need it to be entirely in English and fully functional. Could you provide such a template or guide me on how to adapt the mentioned one? Thank you!

**Reply by windingwind** [reply]
That one is entirely in English, only some descriptions in the post are in other languages. You can directly use it.
